### DISCUSSION TITLE
Recipes section in Wiki

### DISCUSSION THREAD

**Discussion by sloyer** [WebDevStudios/nextjs-wordpress-starter#591] [Ideas]
Hey guys, 

So we are on our 4th site using this great template.
During this time we have come up with quite a few common things that maybe not everyone needs but would be nice to put out to the world to share. 

Examples: 
ACF Repeater blocks (Accordions)
ACF Repeater blocks with images
Querying categories for custom post types
ETC.

It would be great to have somewhere to put these in the wild not apart of the theme but in gists where people can grab the WP code and the Next code and run with it.

**Comment by eric-cortes** [comment]
Do you have an example for the ACF Repeater?

Best,
